workflows/update_tags.sh

```diff
@@ -98,8 +98,22 @@ function update_tags_in_file() {
   sed -i -e "s|${THIS_IMAGE}:.*|${THIS_IMAGE}:${LATEST_TAG}|g" $THIS_FILE
 }
 
+function update_cray_sat_image() {
+  THIS_IMAGE=$1
+  # Capture the output of the podman search for img & update the img if it doesnt exist
+  SEARCH_OUTPUT=$(podman search ${DEFAULT_REGISTRY_NAME}/${THIS_IMAGE} 2>&1)
```

Comment:
I think you should be searching for the existence of the new path, no?

Reply:
Nope, its the other way actually. So the issue was observed when installing CSM 1.6 docs-csm RPM on CSM 1.5 system during the upgrade.

When a docs-csm rpm of 1.6 is tried to install it has the new path were it tries to look up for the tag, but since the nexus is still not updated and the system is 1.5, we should ask it to look at the older path, hence the function looks for the image value passed and since its not found, moves to older path and continues further.

Reply:
Ok, so it is already searching the new path given that's the provided value in the yaml in 1.6.

Reply:
Yep thats correct